src/components/Recipe/Recipe.js

```diff
@@ -0,0 +1,214 @@
+import axios from "axios";
+import { useState, useContext, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import AppContext from "../../context/AppContext";
+import {
+  Box,
+  Button,
+  Grid,
+  Typography,
+  Paper,
+  Table,
+  TableBody,
+  TableContainer,
+  TableHead,
+  TableRow,
+  TableCell,
+} from "@mui/material";
+import RestaurantIcon from "@mui/icons-material/Restaurant";
+
+const Recipe = () => {
+  const [recipe, setRecipe] = useState({});
+  const [isRecipeOpen, setIsRecipeOpen] = useState();
+  const { id } = useParams();
+  const { loggedInUser } = useContext(AppContext);
+  useEffect(() => {
+    // (recipe ? setRecipe({}) : getRecipe(id));
+    const getRecipe = async (id) => {
+      console.log("ID ", id);
+      const response = await axios.get(`/v1/recipes/${id}`);
+      console.log("Response ", response.data);
+      setRecipe(response.data);
+    };
+
+    getRecipe(id);
+  }, [id]);
+
+  let recipe_user_id = recipe.user_id;
+  let isEditable =
+    loggedInUser && recipe_user_id === loggedInUser.id ? true : false;
+  let recipe_user = recipe.user;
```

Comment:
This should also be a const! Do we need recipe_user and reciper_user_id? I think we should just have `recipe_user` and then read `recipe_user.id` when we need it!

Reply:
Also - JS variables should be camel case
```suggestion
  const recipeUser = recipe.user;
```